src/com/carolinarollergirls/scoreboard/defaults/DefaultPenaltyModel.java

```diff
@@ -0,0 +1,27 @@
+package com.carolinarollergirls.scoreboard.defaults;
+
+import com.carolinarollergirls.scoreboard.Skater.Penalty;
+import com.carolinarollergirls.scoreboard.model.SkaterModel.PenaltyModel;
+
+public class DefaultPenaltyModel extends DefaultScoreBoardEventProvider implements PenaltyModel
```

Comment:
Why did you move this out of the SkaterModel? Given PenaltyModel is a subinterface of SkaterModel I'd expect that to stay where it is.